### DISCUSSION TITLE
Feature Request: Add initial time to seeked event

### DISCUSSION THREAD

**Discussion by tobiasrohloff** [vimeo/player.js#677] [Ideas]
The `seeked` event provides the `seconds` the video was seeked to. It would be convenient to also have the initial time position from which the seek was started, e.g. to identify forward and backward seeks.

**Comment by fisherinnovation** [comment]
Thanks for the feedback. We will review this internally and get back to you.

**Comment by tmchow** [comment]
@fisherinnovation Did this every get implemented?
